app/src/main/java/net/gsantner/markor/activity/DocumentShareIntoFragment.java

```diff
@@ -254,7 +247,7 @@ public void onFsViewerConfig(FilesystemViewerData.Options dopt) {
 
                 @Override
                 public void onFsViewerSelected(String request, File file) {
-                    appendToExistingDocument(file, SEP_RULER, true);
+                    appendToExistingDocument(file, _sharedText.length() > 200 ? SEP_RULER : "\n" , true);
```

Comment:
Only add the separator for large text in all caes

Reply:
200 is too small

For example, commonly sharing/bookmarking shopping links - i.e. for this the **link alone without title** is 210 chars 

```
https://www.amazon.de/MOSUO-Temperaturanzeige-Digitalwecker-Einstellbare-Helligkeit/dp/B08C9P9YDC/ref=sr_1_5?__mk_de_DE=%C3%85M%C3%85%C5%BD%C3%95%C3%91&dchild=1&keywords=wanduhr%2Busb&qid=1614904244&sr=8-5&th=1
```

Reply:
200 is too small

For example, when sharing/bookmarking shopping links - the below **link alone without title** is 210 chars 

```
https://www.amazon.de/MOSUO-Temperaturanzeige-Digitalwecker-Einstellbare-Helligkeit/dp/B08C9P9YDC/ref=sr_1_5?__mk_de_DE=%C3%85M%C3%85%C5%BD%C3%95%C3%91&dchild=1&keywords=wanduhr%2Busb&qid=1614904244&sr=8-5&th=1
```

```
[CSL - Nintendo Switch Joy-Con-Aufladehalterung Ladegriff - Wiederaufladbarer 1000 mAh Akku Lithium Ionen - Blaue Status-LED - Komfort-Grip - Für Nintendo Switch Joy Con Gamepad Controller: Amazon.de: Games](https://www.amazon.de/CSL-Joy-Aufladehalterung-Wiederaufladbarer-Komfort-Grip/dp/B07DWD86W5/ref=sr_1_1?__mk_de_DE=%C3%85M%C3%85%C5%BD%C3%95%C3%91&dchild=1&keywords=CSL+-+Nintendo+Switch+Joy-Con-Aufladehalterung+Ladegriff+-+Wiederaufladbarer+1000+mAh+Akku+Lithium+Ionen&qid=1615074679&quartzVehicle=6-1211&replacementKeywords=csl+-+nintendo+switch+joy-con-aufladehalterung+ladegriff+-+wiederaufladbarer+mah+akku+lithium+ionen&sr=8-1 )

$ cat a.txt | wc  -c
643
```

I suggest 1000, what do you think?

Reply:
Actually, I think we should use the tile length alone if it is a link or total length if it isn't